user/dao/user-dao.go

```diff
@@ -53,13 +53,9 @@ func executeQuery(db *sql.DB, query string, args ...interface{}) (int64, error)
 }
 
 // Initialise opens the database connection
-func (dao *DAO) Initialise(configFilePath string) error {
-	config, err := utils.GetConfig(configFilePath)
-	if err != nil {
-		return err
-	}
-
+func (dao *DAO) Initialise(config *utils.Config) error {
 	connStr := fmt.Sprintf("user=%s dbname=%s host=%s sslmode=%s", config.User, config.DBName, config.Host, config.SSLMode)
+	var err error
```

Comment:
This mix of `:=` and `var`+`=` is a bit horrible :′(

Reply:
ikr 😢